rollback.go

```diff
@@ -0,0 +1,56 @@
+package helmclient
+
+import (
+	"context"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/prometheus/client_golang/prometheus"
+	"helm.sh/helm/v3/pkg/action"
+)
+
+// Rollback executes a rollback to a previous revision of a Helm release.
+func (c *Client) Rollback(ctx context.Context, namespace, releaseName string, revision int, options RollbackOptions) error {
+	eventName := "rollback"
+
+	t := prometheus.NewTimer(histogram.WithLabelValues(eventName))
+	defer t.ObserveDuration()
+
+	err := c.rollback(ctx, namespace, releaseName, revision, options)
+	if err != nil {
+		errorGauge.WithLabelValues(eventName).Inc()
+		return microerror.Mask(err)
+	}
```

Comment:
This uses `helm.sh/helm/v3/pkg/action` which is the Helm Go client that was added in Helm 3.

Now our helmclient is mainly a wrapper that does monitoring and logging.